app/src/main/java/com/example/myapplication/adapter/UsersAdapter.kt

```diff
@@ -1,36 +1,78 @@
 package com.example.myapplication.adapter
 
+import android.annotation.SuppressLint
 import android.view.LayoutInflater
+import android.view.View
 import android.view.ViewGroup
 import androidx.recyclerview.widget.DiffUtil
 import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
+import com.example.myapplication.R
 import com.example.myapplication.databinding.UserPatternBinding
 import com.example.myapplication.loadImage
 import com.example.myapplication.model.User
 
+private const val DEFAULT_MARGIN = 8
+private const val MULTISELECT_MODE_MARGIN = 40
 
 class UsersAdapter(private val actionListener: UserActionListener) :
     ListAdapter<User, UsersAdapter.UsersViewHolder>(MyItemCallback()) {
 
+    private var isModeActive: Boolean = false
 
-    class UsersViewHolder(
-        private val binding: UserPatternBinding, private val actionListener: UserActionListener
+    @SuppressLint("NotifyDataSetChanged")
+    fun changeModeStatus(state: Boolean) {
+        isModeActive = state
+        notifyDataSetChanged()
+    }
+
+    inner class UsersViewHolder(
+        private val binding: UserPatternBinding,
     ) : RecyclerView.ViewHolder(binding.root) {
+
         fun onBind(item: User) {
+            val context = binding.root.context
+            val density = context.resources.displayMetrics.density
 
             with(binding) {
+                val layoutParams =
+                    userPhotoView.layoutParams as ViewGroup.MarginLayoutParams
+                root.setBackgroundResource(
+                    if (isModeActive) R.drawable.selected_user_background
+                    else R.drawable.item_background_selector
+                )
+                layoutParams.marginStart =
+                    if (isModeActive) (MULTISELECT_MODE_MARGIN * density).toInt()
+                    else (DEFAULT_MARGIN * density).toInt()
+                userPhotoView.layoutParams = layoutParams
+                deleteUserView.visibility = if (isModeActive) View.GONE else View.VISIBLE
```

Comment:
can be shortened to
deleteUserView.isVisible= !isModeActive

Additionally, I would suggest having 2 separate viewholders and respective item view laoyuts for multiselect mode